src/rmkit/input/events.cpy

```diff
@@ -128,19 +138,14 @@ namespace input:
         case ABS_MT_TRACKING_ID:
           if slot >= 0:
             slots[slot].left = self.left = data.value > -1
-          break
+            if self.left == 0:
+              self.lifted = true
```

Comment:
When we see tracking_id -1 would we want to reset the slot to its initial -1, -1, -1 values? I assume tracking_id=-1 should mean that any existing x,y coords are invalid.

Reply:
if the tracking ID is set to -1, next time the slot is used it will get an X and Y coordinate, i believe.

one potential problem with setting to -1 at this point is that this event is still used (i believe) during mainloop's event dispatch.